core/src/main/java/com/airbnb/aerosolve/core/models/AdditiveModel.java

```diff
@@ -46,13 +76,29 @@ public float debugScoreItem(FeatureVector combinedItem,
         if (func == null)
           continue;
         float val = feature.getValue().floatValue();
-        float subscore = func.evaluate(val);
-        sum += subscore;
+        float subScore = func.evaluate(val);
+        sum += subScore;
         String str = featureFamily.getKey() + ":" + feature.getKey() + "=" + val
-                     + " = " + subscore + "<br>\n";
-        scores.add(new AbstractMap.SimpleEntry<String, Float>(str, subscore));
+                     + " = " + subScore + "<br>\n";
+        scores.add(new AbstractMap.SimpleEntry<String, Float>(str, subScore));
+      }
+    }
+
+    Map<String, List<Double>> denseFeatures = combinedItem.getDenseFeatures();
+    if (denseFeatures != null) {
+      assert (denseWeights != null);
+      for (Map.Entry<String, List<Double>> feature : denseFeatures.entrySet()) {
+        String featureName = feature.getKey();
```

Comment:
to be consistent with other types of features, this is actually feature family name